test/component_test.go

```
@@ -0,0 +1,94 @@
+			bucketArn := atm.Output(component, "vpc_flow_logs_bucket_arn")
+			assert.True(t, strings.HasSuffix(bucketArn, bucketID))
+
+			// Verify that our Bucket has versioning enabled
+			actualStatus := aws.GetS3BucketVersioning(t, awsRegion, bucketID)
+			expectedStatus := "Suspended"
+			assert.Equal(t, expectedStatus, actualStatus)
+
+			policyString := aws.GetS3BucketPolicy(t, awsRegion, bucketID)
+
+			var policy BucketPolicy
+			json.Unmarshal([]byte(policyString), &policy)
```

golangci-lint (1.62.2)

Error return value of `json.Unmarshal` is not checked

(errcheck)